src/fidesops/service/connectors/query_config.py

```diff
@@ -209,7 +209,7 @@ def generate_update_stmt(self, row: Row, policy: Policy) -> Optional[TextClause]
                 f"There is not enough data to generate a valid update statement for {self.node.address}"
             )
             return None
-        query_str = f"UPDATE {self.node.address.collection} SET {','.join(update_clauses)} WHERE  {','.join(pk_clauses)}"
+        query_str = f"UPDATE {self.node.address.collection} SET {','.join(update_clauses)} WHERE  {' AND '.join(pk_clauses)}"
```

Comment:
What are the plans to eventually support masking of PK fields?  This seems like a much trickier problem to not break referential integrity, but one we'll run into, especially on tables with composite PK's.

For example, on the `visit` table in our example postgres database:  there's a composite primary key on `email` and `last_visit`. Currently we can't mask any of this data because they both are PK's, and the statement generated would be something like this because of L204 above: 

```sql
UPDATE visit SET email = 'customer-1@example.com' WHERE  email = 'customer-1@example.com' AND last_visit = '2021-01-06'
```



```yaml
      - name: visit
        fields:
          - name: email
            data_categories: [user.provided.identifiable.contact.email]
            fidesops_meta:
              identity: email
              primary_key: True
          - name: last_visit
            data_categories: [system.operations]
            fidesops_meta:
              primary_key: True
```

Reply:
This seems like it should be a pretty simple fix. Agree that we should add a test for this and a fix.